pkg/tools/log/logruslogger/formatter.go

```diff
@@ -33,6 +34,9 @@ type formatter struct {
 func newFormatter() *formatter {
 	f := formatter{}
 	f.nf.FieldsOrder = []string{"id", "name"}
+	if os.Getenv("LOG_NOCOLORS") == "true" {
+		f.nf.NoColors = true
+	}
```

Comment:
Let's use this behaviour by default. I don't see any problems on our side. So if it's also fine for your side, let's do it.

Reply:
Are you suggesting removing the verification process and adopting the `f.nf.NoColors = true` behavior as the default?

Reply:
Yes, it's correct. I don't see any motivation for using these control symbols so we can just cut it.

Reply:
That would be fine from our side, I will update my commit accordingly.

Reply:
Hello @denis-tingaikin,

I  edited my commit and changed the default behavior based on the discussion. However, some of the CI tests failed, and upon reviewing the results, it seems that some expected outputs explicitly contain the control characters that I have removed.

```
...
--- FAIL: TestOutput (0.00s)
    common_test.go:73: 
        	Error Trace:	/home/runner/work/sdk/sdk/pkg/networkservice/core/trace/traceconcise/common_test.go:73
        	Error:      	Not equal: 
        	            	expected: "\x1b[36m [INFO] [id:conn-1] [type:networkService] \x1b[0mserver-reque...
...
 ```
Do you have any suggestions on how to proceed?

Reply:
Since we're removing control characters from logs, we should also remove them from expectations in tests.